ios/AirMaps/AIRMapUrlTile.h

```diff
@@ -0,0 +1,35 @@
+//
+//  AIRUrlTileOverlay.h
+//  AirMaps
+//
+//  Created by cascadian on 3/19/16.
+//  Copyright © 2016. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <MapKit/MapKit.h>
+#import <UIKit/UIKit.h>
+
+#import "RCTConvert+MapKit.h"
+#import "RCTComponent.h"
+#import "AIRMapCoordinate.h"
+#import "AIRMap.h"
+#import "RCTView.h"
+
+@interface AIRMapUrlTile : MKAnnotationView <MKOverlay>
```

Comment:
And actually, maybe this is the solution... Why can't this class just be a subclass of `NSObject` that conforms to `MKOverlay`? That seems to be all that is required to make this work. I can't tell why this being a subclass of `MKAnnotationView` is actually important.